hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -257,6 +257,7 @@ public static HoodieTableMetaClient initTableIfNotExists(Configuration conf) thr
           .setTableCreateSchema(conf.getString(FlinkOptions.SOURCE_AVRO_SCHEMA))
           .setTableType(conf.getString(FlinkOptions.TABLE_TYPE))
           .setTableName(conf.getString(FlinkOptions.TABLE_NAME))
+          .setDatabaseName(conf.getString(FlinkOptions.DATABASE_NAME))
```

Comment:
side effects?

Reply:
The table service uses db_name.tbl_name to determine the unique hoodie table, but currently Flink does not set it by default db_name.